docs/notebooks/0_introduction.ipynb

```diff
@@ -531,7 +427,6 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The metadata of a given data product is the information provided by the user on the upload form. This information is attached to the data product contents and is available for consulting on the PZ Server page or using this Python API (`pzserver`). \n",
     "The metadata of a given data product is the information provided by the user on the upload form. This information is attached to the data product contents and is available for consulting on the PZ Server page or using this Python API (`pzserver`). \n",
     "\n",
     "All data products stored on PZ Server are identified by a unique **id** number or an unique name, a _string_ called **internal_name**, which is created automatically at the moment of the upload by concatenating the product **id** to the name given by its owner (replacing blank spaces by \"_\", lowering cases, and removing special characters). "
```

Comment:
Despite starting with a vowel, the "u" in unique has a consonant sound (something like "you"). So, the correct article is "a".

Below, in  "The time spent to download a data product depends on the internet connections between the user and the host", I think it should be just "connection".